subworkflows/nf-core/vcf_impute_glimpse/meta.yml

```diff
@@ -0,0 +1,94 @@
+name: "vcf_imputation_glimpse"
+description: Impute VCF/BCF files with Glimpse
+keywords:
+  - glimpse
+  - chunk
+  - phase
+  - ligate
+
+modules:
+  - glimpse/chunk
+  - glimpse/phase
+  - glimpse/ligate
+
+input:
```

Comment:
Inputs in subworkflows should be seperated per input channel. You can describe in the descriptions of each channel what files should be in it (preferable with the structure), see here for an example: https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/vcf_extract_relate_somalier/meta.yml